src/dev-app/index.html

```diff
@@ -25,13 +25,30 @@
 <body>
   <dev-app>Loading...</dev-app>
 
+  <iframe id="google-maps-api-key"
+          src="google-maps-api-key.txt"
+          style="display:none;"
+          onload="loadGoogleMapsScript()"></iframe>
+
   <script src="core-js-bundle/index.js"></script>
   <script src="zone.js/dist/zone.js"></script>
   <script src="systemjs/dist/system.js"></script>
   <script src="system-config.js"></script>
   <script src="https://www.youtube.com/iframe_api"></script>
-  <script src="https://maps.googleapis.com/maps/api/js"></script>
   <script src="https://unpkg.com/@googlemaps/markerclustererplus/dist/index.min.js"></script>
+  <script>
+    function loadGoogleMapsScript() {
+      var iframe = document.getElementById('google-maps-api-key');
+      var googleMapsScript = document.createElement('script');
+      var googleMapsApiKey = iframe.contentDocument.body.textContent;
+      var googleMapsUrl = 'https://maps.googleapis.com/maps/api/js';
+      if (googleMapsApiKey !== 'Page not found') {
+        googleMapsUrl = googleMapsUrl + '?key=' + googleMapsApiKey;
```

Comment:
I'd have to look into it, but there should be a cleaner way of loading this.

Reply:
Yeah, I'm not crazy about this either. I spent a few hours trying to come up with something better, but this was the best I was able to come up with. Let me know if you have something else you want me to try.

Reply:
One idea: could we have the webserver read the key from an environment variable and append it into the `index.html` when it's served?

Reply:
Our dev server doesn't look like it's set up to alter the html it's serving. We'd need to be serving jade templates or something I think. So, I still haven't found a better way of doing this. For now I'll add the comments suggested below, and if we find a better way of hiding the API key then I'll do that in another pr.